lib/peer.js

```diff
@@ -33,28 +30,46 @@ var MAX_RECEIVE_BUFFER = 10000000;
  * @returns {Peer} A new instance of Peer.
  * @constructor
  */
-function Peer(host, port, network, relay) {
```

Comment:
The new opts constructor is an improvement, but not a backwards-compatible one. Probably should bump the version to 0.11 for merge.